src/app/core/model/partner.ts

```diff
@@ -0,0 +1,2 @@
+export class Partner {
```

Comment:
You are not using this model. You should do it. It should be an interface, since this is a defined object. It also should extend DataType (shared/models/data-type).

Reply:
Also, add "model" to the file name: partner.model.ts